minio/minio
# feat: Encrypt s3zip file index

## Description

When the object is encrypted on disk also encrypt the zip index.

For these `x-minio-internal-archive-type` will be set to "zip-enc", so fully backwards compatible. The same plaintext is used as base string.

## Motivation and Context

File names inside zip files should be encrypted on disk.

## How to test this PR?

Use s3zip functionality. For example `warp zip`

`"x-minio-internal-archive-type": "emlwLWVuYw=="`  (raw: "zip-enc") should be set in metadata.

Mint also tests s3zip functionality.

## Types of changes
